app/controllers/api/v2/template_controller.rb

```diff
@@ -37,7 +37,14 @@ def import
       param_group :taxonomies, ::Api::V2::BaseController
       def export
         @result = ForemanTemplates::TemplateExporter.new(template_export_params).export!
-        render :json => { :message => @result.to_h }, :status => @result.exported ? 200 : 500
+        render :json => { :message => { :error => @result[:error],
```

Comment:
Would it make sense to have a custom object that would take care of constructing the response shape so that we do not have this in controller?

Reply:
@xprazak2, I'm not sure if there is a need in specific object, which will require more refactoring if we want to make it as general as it allows to reuse within `import` as well as in UI/API responses for each.